src/messages/MMonCommand.h

```diff
@@ -55,22 +59,27 @@ class MMonCommand final : public PaxosServiceMessage {
     std::ostringstream ss;
     std::string prefix;
     cmdmap_from_json(cmd, &cmdmap, ss);
-    cmd_getval(cmdmap, "prefix", prefix);
-    // Some config values contain sensitive data, so don't log them
-    o << "mon_command(";
-    if (prefix == "config set") {
-      std::string name;
-      cmd_getval(cmdmap, "name", name);
-      o << "[{prefix=" << prefix << ", name=" << name << "}]";
-    } else if (prefix == "config-key set") {
-      std::string key;
-      cmd_getval(cmdmap, "key", key);
-      o << "[{prefix=" << prefix << ", key=" << key << "}]";
-    } else {
-      for (unsigned i=0; i<cmd.size(); i++) {
-        if (i) o << ' ';
-        o << cmd[i];
+    try {
+      cmd_getval(cmdmap, "prefix", prefix);
+      // Some config values contain sensitive data, so don't log them
+      o << "mon_command(";
+      if (prefix == "config set") {
+        std::string name;
+        cmd_getval(cmdmap, "name", name);
+        o << "[{prefix=" << prefix << ", name=" << name << "}]";
+      } else if (prefix == "config-key set") {
+        std::string key;
+        cmd_getval(cmdmap, "key", key);
+        o << "[{prefix=" << prefix << ", key=" << key << "}]";
+      } else {
+        for (unsigned i=0; i<cmd.size(); i++) {
+          if (i) o << ' ';
+          o << cmd[i];
+        }
       }
+    } catch (bad_cmd_get& e) {
```

Comment:
OK, so most of the changes in this commit is about reindenting `cmd_getval()`-related blocks due to catching `bad_cmd_get`.